test/spec/app_spec_helpers.rb

```diff
@@ -228,4 +228,61 @@ def save_screenshot(title)
     uploaded = client.upload(image)
     uploaded.link
   end
-end
+
+  def wait_page_load(options = {})
+    driver = options[:driver] || @driver
+    item = options[:item] || "root"
+    @wait.until {driver.page_source.include?(item) }
+  end
+
+  def go(new_url)
+    if defined? $caller_name and $caller_name.length > 0
+      method_id = $caller_name[0]
+      method_id.gsub! (/(\s)/), '_'
+      method_id.gsub! (/("|\[|\])/), ''
+      if new_url.include? '?'
+        new_url = new_url + '&test_id='+method_id
+      else
+        new_url = new_url + '?test_id='+method_id
+      end
+    end
+    @driver.navigate.to new_url
+  end
+
+
+  def new_driver(webdriver_url, browser_capabilities)
+    if @config.key?('proxy') and !webdriver_url.include? "browserstack"
+      proxy = Selenium::WebDriver::Proxy.new(
+        :http     => @config['proxy'],
+        :ftp      => @config['proxy'],
+        :ssl      => @config['proxy']
+      )
+      if (Dir.entries(".").include? "extension.crx")
+        caps = Selenium::WebDriver::Remote::Capabilities.chrome ({
+            'chromeOptions' => {
+              'extensions' => [
+                Base64.strict_encode64(File.open('./extension.crx', 'rb').read)
+              ]
+            }, :proxy => proxy
+          })
+      else
+        caps = Selenium::WebDriver::Remote::Capabilities.chrome(:proxy => proxy)
+      end
+      dr = Selenium::WebDriver.for(:chrome, :desired_capabilities => caps , :url => @config['chromedriver_url'])
```

Comment:
Instead of  `@config['chromedriver_url']` it should be the `webdriver_url` passed as parameter, no?

Reply:
Yes, since we don't have `quick_spec.rb` anymore